frontend/src/pages/api/status-indexer.js

```diff
@@ -0,0 +1,27 @@
+import { getNearNetwork } from "../../libraries/config";
+import { ExplorerApi } from "../../libraries/explorer-wamp";
+import BlocksApi from "../../libraries/explorer-wamp/blocks";
+
+export default async function (req, res) {
+  // This endpoint provides the status of Indexer for Explorer
+  // It checks the latest block height in indexer database
+  // and the latest block height from JSON RPC
+  // If the difference in between 30 blocks it is considered as fine
+  // otherwise not fine
+
+  try {
+    const rpcFinalBlock = await new ExplorerApi(req).call(
+      "nearcore-final-block"
+    );
+    const indexerFinalBlock = await new BlocksApi(req).getBlocks(1, null);
+
+    const statusResponse = {
+      rpc_latest_block_height: rpcFinalBlock.header.height,
+      indexer_latest_block_height: indexerFinalBlock[0].height,
```

Comment:
Maybe it's better to pass timestamp? (could be in addition to height)
It will allow showing the time lag (smth like "we are late for 25 minutes")

Reply:
It's not a problem yet not sure it's useful. @frol what do you think?

Reply:
At least for now, I don't see a use for it, so I would not spend cycles here. If we observe a need, it will be trivial to add